### PR TITLE
[CFM-1078] Adding the radio button when a user is a member of a project.

### PR DESCRIPTION
**Pull request by ordavidil**
https://issuetracker.amplexor.com/jira/browse/CFM-1078

### CODE REVIEW

**Comment by ordavidil** [comment]
@soniCaH 
Can't continue since it seems the functionality doesn't work on __my groups__ as well

**Comment by soniCaH** [comment]
@ordavidil #1006 was merged.

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module [hunk 1]
```diff
@@ -1232,9 +1232,13 @@ function c4m_message_load_homepage_activity_stream() {
 
   if ($user->uid) {
     // This is an Authenticated user.
-    if ($filter == 'interests' || $filter == 'groups') {
+    if ($filter == 'interests' || $filter == 'groups' || $filter == 'projects') {
       $groups = c4m_og_get_user_group_nodes_by_bundle('group');
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module]: @guytepper 
Let's use the `switch` as discussed yesterday so we won't hit the DB if not needed ...

**Comment by guytepper** [comment]
@ordavidil ready for review.

**Pull request merged**